### PR TITLE
feat: Add how to use pgvecto.rs in Kubernetes

### PR DESCRIPTION
**Pull request by xieydd**
fix https://github.com/tensorchord/pgvecto.rs/issues/211#issuecomment-1882224988

### CODE REVIEW

**Comment by gaocegege** [comment]
The deployment failed. PTAL

**Comment by xieydd** [comment]
> The deployment failed. PTAL

Fixed, PTAL.

**Comment by gaocegege** [comment]
LGTM

cc @VoVAllen

File: src/admin/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+If you want to use `pgvecto.rs` in PostgreSQL which is running in Kubernetes, we provide a tutorial to help you. In this tutorial, we will use [CloudNative-PG](https://cloudnative-pg.io/) to deploy PostgreSQL cluster in Kubernetes.
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_helm"></a> [helm](#requirement\_helm) | >= 2.4.1 |
+| <a name="requirement_kubectl"></a> [kubectl](#requirement\_kubectl) | >= 1.14 |
+| <a name="requirement_kubernetes"></a> [kubernetes](#requirement\_kubernetes) | >= 2.10 |
```
Comment (bo0tzz) [inline src/admin/kubernetes.md:11]: There is no kubernetes version `2.10`. The earliest kubernetes version supported by cloudnative-pg is `1.24.0`.

File: src/admin/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+## Helm Instal CloudNative-PG
```
Comment (bo0tzz) [inline src/admin/kubernetes.md:13]:
```suggestion
## Helm Install CloudNative-PG
```

File: src/admin/kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+ADD https://github.com/tensorchord/pgvecto.rs/releases/download/$PGVECTORS_TAG/vectors-pg${CNPG_TAG%.*}_${PGVECTORS_TAG#"v"}_$TARGETARCH.deb ./pgvectors.deb
+RUN apt install ./pgvectors.deb
+
+USER postgres
+```
+
+```shell
+$ docker build -t <your-postgresql-image-repo>:15 --build-arg CNPG_TAG=15 --build-arg PGVECTORS_TAG=v0.1.13   --build-arg TARGETARCH=amd64 .
+```
+> Notice: PostgreSQL docker image tag must contain CNPG_TAG.
```
Comment (bo0tzz) [inline src/admin/kubernetes.md:53]:
```suggestion
> Notice: PostgreSQL docker image tag must start with CNPG_TAG for cloudnative-pg to recognize the postgres version.
```

**Review by bo0tzz — COMMENTED** [review]
Lovely stuff!

If you want to provide more official support for cloudnative-pg, I would be happy to transfer my https://github.com/bo0tzz/cnpgvecto.rs repository into your organization and help maintain it there.

File: src/admin/kubernetes.md (hunk 1 above)
Comment (xieydd) [reply src/admin/kubernetes.md:11]: Thanks a lot, I fixed it.

File: src/admin/kubernetes.md (hunk 2 above)
Comment (xieydd) [reply src/admin/kubernetes.md:13]: tks.

File: src/admin/kubernetes.md (hunk 3 above)
Comment (xieydd) [reply src/admin/kubernetes.md:53]: tks.

**Comment by xieydd** [comment]
> Lovely stuff!
> 
> If you want to provide more official support for cloudnative-pg, I would be happy to transfer my https://github.com/bo0tzz/cnpgvecto.rs repository into your organization and help maintain it there.

Appreciate your work @bo0tzz . Can we provide a repository for more official support? @gaocegege @VoVAllen

**Comment by gaocegege** [comment]
@bo0tzz Thanks and welcome!

I invited you into the team tensorchord/pgvecto-rs-maintainers.

Please visit https://github.com/tensorchord to accept the invitation. Here is the repository: https://github.com/tensorchord/cloudnative-pgvecto.rs

**Comment by bo0tzz** [comment]
Thank you! I've accepted the invite :)

To start, should I just push the content of bo0tzz/cnpgvecto.rs into this new repository?

**Comment by xieydd** [comment]
> Thank you! I've accepted the invite :)
> 
> To start, should I just push the content of bo0tzz/cnpgvecto.rs into this new repository?

Yep, you can push the content of bo0tzz/cnpgvecto.rs into the main branch of the new repo.

**Comment by xieydd** [comment]
@gaocegege @VoVAllen Can we merge this pr?

**Pull request merged**